packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.tsx

```diff
@@ -96,16 +96,20 @@ export class SearchBox extends BaseComponent<ISearchBoxProps, ISearchBoxState> {
   }
 
   @autobind
-  private _onClearClick(ev?: any) {
-    this._latestValue = '';
-    this.setState({
-      value: ''
-    });
-    this._callOnChange('');
-    ev.stopPropagation();
-    ev.preventDefault();
+  private _onClear(ev?: any) {
```

Comment:
This shouldn't be any. Can you remove all the anys? Instead use HTMLEvent or some variant of that.

Reply:
Can do.

Reply:
and done